sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -186,3 +191,143 @@ public partial class TableResponseProperties
+        public string Version { get { throw null; } }
+        protected static Azure.Data.Tables.Sas.SasQueryParameters Create(System.Collections.Generic.IDictionary<string, string> values) { throw null; }
+        protected static Azure.Data.Tables.Sas.SasQueryParameters Create(string version, Azure.Data.Tables.Sas.AccountSasServices? services, Azure.Data.Tables.Sas.AccountSasResourceTypes? resourceTypes, Azure.Data.Tables.Sas.SasProtocol protocol, System.DateTimeOffset startsOn, System.DateTimeOffset expiresOn, Azure.Data.Tables.Sas.SasIPRange ipRange, string identifier, string resource, string permissions, string signature, string cacheControl = null, string contentDisposition = null, string contentEncoding = null, string contentLanguage = null, string contentType = null) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class TableSasBuilder
+    {
+        public TableSasBuilder(string tableName) { }
+        public System.DateTimeOffset ExpiresOn { get { throw null; } set { } }
+        public string Identifier { get { throw null; } set { } }
+        public Azure.Data.Tables.Sas.SasIPRange IPRange { get { throw null; } set { } }
+        public string PartitionKeyEnd { get { throw null; } set { } }
+        public string PartitionKeyStart { get { throw null; } set { } }
+        public string Permissions { get { throw null; } }
+        public Azure.Data.Tables.Sas.SasProtocol Protocol { get { throw null; } set { } }
+        public string RowKeyEnd { get { throw null; } set { } }
+        public string RowKeyStart { get { throw null; } set { } }
+        public System.DateTimeOffset StartsOn { get { throw null; } set { } }
+        public string TableName { get { throw null; } }
+        public string Version { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public void SetPermissions(Azure.Data.Tables.Sas.TableSasPermissions permissions) { }
+        public void SetPermissions(string rawPermissions) { }
+        public Azure.Data.Tables.Sas.TableSasQueryParameters ToSasQueryParameters(Azure.Data.Tables.TablesSharedKeyCredential sharedKeyCredential) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override string ToString() { throw null; }
+    }
+    [System.FlagsAttribute]
+    public enum TableSasPermissions
+    {
+        All = -1,
+        Read = 1,
+        Add = 2,
+        Update = 4,
+        Delete = 8,
+    }
+    public sealed partial class TableSasQueryParameters : Azure.Data.Tables.Sas.SasQueryParameters
```

Comment:
Do we need this type?

Reply:
yes, it differentiates the query params unique to tables as distinct from account.